lib/contributors/prompt.js

```diff
@@ -29,17 +29,23 @@ function getQuestions(options, username, contributions) {
     name: 'contributions',
     message: 'What are the contribution types?',
     when: !contributions,
+    default: function (answers) {
+      // default values for contributions when updating existing users
+      return allContributions
+        .filter((entry) => entry.login.toLowerCase() === answers.username.toLowerCase())
+        .reduce((memo, entry) => memo.concat(entry.contributions), []);
```

Comment:
What does `memo` stands for?

Reply:
It's a fairly common argument name for the first argument in a reducer.

Reply:
Thank you for the explanation! Good for me as well then :)